<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hotelmanagerdialog</class>
 <widget class="QDialog" name="hotelmanagerdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0.522, y1:1, x2:0.497, y2:0, stop:0 rgba(190, 231, 243, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>631</width>
     <height>501</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>酒店信息管理</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>621</width>
       <height>381</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>基本信息</string>
      </attribute>
      <widget class="QLineEdit" name="address">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>100</y>
         <width>391</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="hotelname">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>30</y>
         <width>391</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_9">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>酒店地址</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_28">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>170</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>酒店电话</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="dial">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>170</y>
         <width>391</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_29">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>酒店名称</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_30">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>240</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>房间介绍</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="discription">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>240</y>
         <width>391</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>定价</string>
      </attribute>
      <widget class="QLineEdit" name="pdbl">
       <property name="geometry">
        <rect>
         <x>461</x>
         <y>80</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>22</x>
         <y>80</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>双人标间</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>321</x>
         <y>80</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>单价/间/天</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="dblnum">
       <property name="geometry">
        <rect>
         <x>161</x>
         <y>80</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>22</x>
         <y>200</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>豪华套房</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>321</x>
         <y>30</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>单价/间/天</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="pbbd">
       <property name="geometry">
        <rect>
         <x>461</x>
         <y>140</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>22</x>
         <y>30</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>单人标间</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="sglnum">
       <property name="geometry">
        <rect>
         <x>161</x>
         <y>30</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="bbdnum">
       <property name="geometry">
        <rect>
         <x>161</x>
         <y>140</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="psuite">
       <property name="geometry">
        <rect>
         <x>461</x>
         <y>200</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>22</x>
         <y>140</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>大床房</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="suitenum">
       <property name="geometry">
        <rect>
         <x>161</x>
         <y>200</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="psgl">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>30</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>321</x>
         <y>140</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>单价/间/天</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>321</x>
         <y>200</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>单价/间/天</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="discount">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>260</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_25">
       <property name="geometry">
        <rect>
         <x>21</x>
         <y>260</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>折扣</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>上传图片</string>
      </attribute>
      <widget class="QLineEdit" name="img_out">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>60</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="chooseout">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>60</y>
         <width>93</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>浏览</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>24</y>
         <width>72</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>外景图片</string>
       </property>
      </widget>
      <widget class="QPushButton" name="choosein1">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>150</y>
         <width>93</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>浏览</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>114</y>
         <width>72</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>内景图片1</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="img_in01">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>150</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="choosein2">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>240</y>
         <width>93</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>浏览</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>204</y>
         <width>72</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>内景图片2</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="img_in02">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>240</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="updateimg">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>300</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>上传图片</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>修改</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>20</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>提交修改</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exit">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>430</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>订单管理</string>
    </attribute>
    <widget class="QTableWidget" name="ordertable">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>591</width>
       <height>281</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>用户名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>性别</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>联系电话</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>房间类型</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>入住天数</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>入住日期</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>房间数</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>订单总金额</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>订单状态</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="changestatus">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>380</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>修改订单状态</string>
     </property>
    </widget>
    <widget class="QPushButton" name="refresh">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>380</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>刷新</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>281</width>
       <height>281</height>
      </rect>
     </property>
     <property name="title">
      <string>订单状态修改</string>
     </property>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>72</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>用户名</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Username">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>40</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>72</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>订单号</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="No">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>80</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>72</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>总金额</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Total">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>120</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>160</y>
        <width>72</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>订单状态</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>160</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>已下单</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>已支付</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>已退款</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>已入住</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>已完成</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="save">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>220</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="exit_2">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>380</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
